HW#4 -- Semenova Anastasia

a-smnv opened the pull request.

jeanny-driver commented on HW#4.js:
```diff
@@ -0,0 +1,70 @@
+
+// 3
+const toCamelCase = str => str.split(/-|_/)
+															.map((word, i) => i === 0 ? word
```
![image](https://user-images.githubusercontent.com/71435688/98001149-5cc14200-1dfe-11eb-8592-67ac22e5c7e0.png)
 If it's really so, the simplest way is to Format Document (right click in js file).

jeanny-driver reviewed (commented):
Well done!!! 💎

a-smnv commented:
Hello,
thank you for the feedback
I'll format the document.)

Semenova Anastasia
т. +7-964-374-29-98




вт, 3 нояб. 2020 г. в 18:04, jeanny-driver <notifications@github.com>:

> *@jeanny-driver* commented on this pull request.
>
> Well done!!! 💎
> ------------------------------
>
> In HW#4.js
> <https://github.com/js-training-sep-2020/hw-4-js-functions-with-es6/pull/11#discussion_r516730845>
> :
>
> > @@ -0,0 +1,70 @@
>
> +// 1
>
> +const splitAndMerge = (str, joiner) => str.split(" ")
>
> +																					.map((word) => word.split("").join(joiner))
>
> +																					.join(" ");
>
> +console.log(splitAndMerge("My name is John"," ")); //"M y n a m e i s J o h n"
>
> +console.log(splitAndMerge("Hello World!",",")); //"H,e,l,l,o W,o,r,l,d,!"
>
> +
>
> +// 2
>
> +const convert = obj => Object.entries(obj);
>
> +console.log(convert({ name: "Jeremy", age: 24, role: "Software Engineer" })); //[["name", "Jeremy"], ["age", 24], ["role", "Software Engineer"]]
>
> +
>
> +// 3
>
> +const toCamelCase = str => str.split(/-|_/)
>
> +															.map((word, i) => i === 0 ? word
>
>
> I suppose there are some issue with Formatting... Or maybe it's smth wrong
> from my side 😅
> [image: image]
> <https://user-images.githubusercontent.com/71435688/97999993-0bfd1980-1dfd-11eb-838d-c6f9aa53f503.png>
> [image: image]
> <https://user-images.githubusercontent.com/71435688/98000030-161f1800-1dfd-11eb-8a08-32435260a530.png>
> ------------------------------
>
> In HW#4.js
> <https://github.com/js-training-sep-2020/hw-4-js-functions-with-es6/pull/11#discussion_r516732496>
> :
>
> > @@ -0,0 +1,70 @@
>
> +// 1
>
> +const splitAndMerge = (str, joiner) => str.split(" ")
>
> +																					.map((word) => word.split("").join(joiner))
>
> +																					.join(" ");
>
> +console.log(splitAndMerge("My name is John"," ")); //"M y n a m e i s J o h n"
>
> +console.log(splitAndMerge("Hello World!",",")); //"H,e,l,l,o W,o,r,l,d,!"
>
> +
>
> +// 2
>
> +const convert = obj => Object.entries(obj);
>
> +console.log(convert({ name: "Jeremy", age: 24, role: "Software Engineer" })); //[["name", "Jeremy"], ["age", 24], ["role", "Software Engineer"]]
>
> +
>
> +// 3
>
> +const toCamelCase = str => str.split(/-|_/)
>
> +															.map((word, i) => i === 0 ? word
>
>
> [image: image]
> <https://user-images.githubusercontent.com/71435688/98001149-5cc14200-1dfe-11eb-8592-67ac22e5c7e0.png>
> If it's really so, the simplest way is to Format Document (right click in
> js file).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/js-training-sep-2020/hw-4-js-functions-with-es6/pull/11#pullrequestreview-522581971>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALVWM45NQ4F5L5NUDQFJ6GTSOALXPANCNFSM4S7XQSPQ>
> .
>
